libpod/image/image.go

```diff
@@ -1475,7 +1476,8 @@ func (i *Image) Save(ctx context.Context, source, format, output string, moreTag
 			return errors.Wrapf(err, "error getting OCI archive ImageReference for (%q, %q)", output, destImageName)
 		}
 	case "oci-dir":
-		destRef, err = directory.NewReference(output)
+		destImageName := imageNameForSaveDestination(i, source)
+		destRef, err = layout.NewReference(output, destImageName)
 		if err != nil {
 			return errors.Wrapf(err, "error getting directory ImageReference for %q", output)
 		}
```

Comment:
BTW below, calling the format `docker-dir` is somewhat misleading: It’s a c/image-invented format. I don’t care about the misattribution, just noting that some people are interested to avoiding certain words :) , so maybe it might be worth adding an alias.

Reply:
This one we will need to let go.